parsec/core/backend_connection/authenticated.py

```diff
@@ -359,8 +379,8 @@ async def _manager_connect(self):
                     user_profile_outsider_allowed=rep.user_profile_outsider_allowed,
                     active_users_limit=rep.active_users_limit,
                     # Sequester introduced in APIv2.8/3.2
-                    sequester_authority=rep.sequester_authority_certificate,
-                    sequester_services=rep.sequester_services_certificates,
+                    sequester_authority=rep.sequester_authority_certificate,  # type: ignore[arg-type]
+                    sequester_services=rep.sequester_services_certificates,  # type: ignore[arg-type]
```

Comment:
Hmmm is this a bug in the code base?

Reply:
It looks like there's a bug here the rust binding returns `bytes` for `sequester_authority_certificate` and `sequester_services_certificates` but the python `OrganizationConfig` object which stores these values expects `Optional[SequesterAuthorityCertificate]` and `Optional[Tuple[SequesterServiceCertificate, ...]]`. Is it an oxidation mistake?

Reply:
@touilleMan what's the proper fix for this?